docs/officers/dues/data-model.rst

```diff
@@ -0,0 +1,49 @@
+Data Model
+==========
+
+Dues are tracked in Apiary using several related objects.
+
+-----------
+Fiscal Year
+-----------
+
+A **Fiscal Year** groups together several `Dues Package`_ and `Merchandise`_ objects.
+Fiscal years are primarily used for reporting and do not affect any functionality.
+They need to be created manually once a year to group the objects for that year.
+
+---------------
+_`Dues Package`
+---------------
+
+A **Dues Package** represents an option for paying dues.
+In the member-facing interface, packages are labeled as "Dues Terms."
+There are a few key attributes:
+
+- The **cost** of the package
+- The **membership start date**, when someone is considered to be an active member if they purchase this package
+- The **membership end date**, when someone is no longer considered to be an active member unless/until they purchase another package
+- The **access start date**, when someone will gain access to RoboJackets systems if they purchase this package
+- The **access end date**, when someone will lose access to RoboJackets systems unless/until they purchase another package
```

Comment:
```suggestion
- The **membership start date**, when someone is considered to be an active member if they purchase this package (generally the start of the fall or spring semester)
- The **membership end date**, when someone is no longer considered to be an active member unless/until they purchase another package (generally the end of the summer or fall semester)
- The **access start date**, when someone will gain access to RoboJackets systems if they purchase this package (generally whenever the package is first opened for payments)
- The **access end date**, when someone will lose access to RoboJackets systems unless/until they purchase another package (generally the dues deadline for the following term)
```

I'm sure there's a better way to add this info but I don't know what it is.

Reply:
I see the instructions page now, so this suggestion might be irrelevant. It might still be useful here though.